examples/hybrid/staggered_nonhydrostatic_model.jl

```diff
@@ -166,6 +166,9 @@ function implicit_tendency_special!(Yₜ, Y, p, t)
 
     ref_thermo_params = Ref(thermo_params)
     ref_zuₕ = Ref(zero(eltype(Yₜ.c.uₕ)))
+    FT = eltype(Yₜ)
```

Comment:
I don't think that this is how you get the float type

Reply:
is this better `FT = eltype(Yₜ.c.uₕ)`

Reply:
I think `FT` is already defined. Seems like all the other functions are just magically using it in this file